shipyard-controller/go.mod

```diff
@@ -17,6 +17,9 @@ require (
 	github.com/swaggo/swag v1.7.8
 	github.com/tryvium-travels/memongo v0.3.2
 	go.mongodb.org/mongo-driver v1.7.5
+	go.uber.org/zap v1.13.0 // indirect
```

Comment:
not sure if these additional entries are intended - can you try to run `go mod tidy` and see if that removes them?

Reply:
executed, nothing changed, I have not added anything manually, just executed `go mod tidy`